browser/components/preferences/in-content/main.js

```diff
@@ -28,7 +28,7 @@ var gMainPane = {
       document.getElementById(aId)
               .addEventListener(aEventType, aCallback.bind(gMainPane));
     }
-    this.updateLocale();
+    this.getDefaultLocal();
```

Comment:
Sets language selector to current locale value without `updateLocale` notification.